src/graph/RaytraceNode.cpp

```diff
@@ -22,6 +22,8 @@
 
 void RaytraceNode::validate()
 {
+	setFields(findFieldsToCompute());
+
```

Comment:
It would be good to add a comment that it should be viewed as a temporary solution.

Reply:
Also - I'd need help to solve it in v14. `validate()` is replaced with a few functions:
- `onInputChange()`
- `onGraphChange()`
None is always called before execution as `validate()` was; they are meant to be called only when X changes. I could imagine a similar method for handling changing field requirements. However, it might get messy and tricky. Instead, I'd propose to move this burden onto user.

Reply:
Sure. Let's discuss this offline.